src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -184,6 +189,21 @@ func (l *commitLog) Open() error {
 	return nil
 }
 
+func (l *commitLog) ActiveLogs() ([]File, error) {
```

Comment:
Why have this function return `[]File`? It seems like you return at most one `File`.

Reply:
@prateek Was talking about having support for multiple commit log files at some point